README.md

```diff
@@ -101,34 +101,34 @@ via this installation procedure.
 ## FAQ and Troubleshooting
 
 1. ```deepchem``` currently supports both Python 2.7 and Python 3.5, and is supported on 64 bit Linux and Mac OSX. Note that DeepChem is not currently maintained for Python 3.6 or with other operating systems. 
-2. Question: I'm seeing some failures in my test suite having to do with MKL
+2. Question: I see some failures in my test suite having to do with MKL

```

Comment:
The original is more appropriate keeping in mind it's present continuous tense nature